### PR TITLE
Changed variable type LockPropertyType to StringProperty in GWTPropertyDescriptor

### PR DESCRIPTION
**Pull request by labkey-bpatel**
To avoid serialization issues on teamcity postgres.

### CODE REVIEW

**Comment by labkey-tchad** [comment]
Waiting for BVTs and Issues suites to pass on [TeamCity](https://teamcity.labkey.org/viewLog.html?buildId=771448&buildTypeId=bt2&tab=dependencies&branch_LabkeyTrunk=lockedPropertyRefactor#_expand=block_bt10103_bt10104_bt10105_bt10106_bt1555_bt1556_bt401_bt402_bt488_bt489_bt5436_bt5437_bt5468_bt5469_bt5496_bt5497_bt5499_bt5500_bt5501_bt5502_bt7_bt8_bt8683_bt9904_bt9905-771458_771527_771532&hpos=0&vpos=1037)

**Comment by labkey-tchad** [comment]
TeamCity results look good

File: api/gwtsrc/org/labkey/api/gwt/client/LockedPropertyType.java [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+
+    LockedPropertyType(String label)
+    {
+        _label = label;
+    }
 
-    LockedPropertyType() {}
+    public String getLabel()
+    {
+        return _label;
```
Comment (labkey-jeckels) [inline api/gwtsrc/org/labkey/api/gwt/client/LockedPropertyType.java]: Should be no need for label in this enum

Comment (labkey-bpatel) [reply api/gwtsrc/org/labkey/api/gwt/client/LockedPropertyType.java]: Done.

**Pull request merged**